js/src/block-editor/index.js

```diff
@@ -3,15 +3,29 @@
 /**
  * WordPress dependencies
  */
-import { setLocaleData } from '@wordpress/i18n';
+import domReady from '@wordpress/dom-ready';
 import { addFilter } from '@wordpress/hooks';
+import { setLocaleData } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { addControls, registerBlocks } from './helpers';
 import { Edit } from './components';
+import { GAClient } from '../common/classes';
 
 setLocaleData( { '': {} }, 'genesis-custom-blocks' );
 addFilter( 'genesisCustomBlocks.controls', 'genesisCustomBlocks/addControls', addControls );
+
+// @ts-ignore
 registerBlocks( genesisCustomBlocks, gcbBlocks, Edit );
+
+domReady( () => {
+	// @ts-ignore
+	window.dataLayer = window.dataLayer || [];
```

Comment:
What was you logic for moving `window.dataLayer` our of `GAClient` ?

Reply:
Not a functional reason, I just like class files to have only the class, and no logic other than that.

Maybe from too much time in PHP 🤣 

But if you'd prefer, I could move it back.

Reply:
Actually, I'm going to move it back.

Reply:
Moved it back in https://github.com/studiopress/genesis-custom-blocks/pull/101/commits/9d8bd80bc6dc9781f6f2fdb8f24ece00ed3408d3